test/loadmodels.jl

```diff
@@ -247,6 +247,23 @@ end
     ]
 end
 
+@testset "Constraints" begin
+    m = readSBML(joinpath(@__DIR__, "data", "custom.xml"))
+    @test only(m.constraints) == SBML.Constraint(
+    SBML.MathApply("and", [
+        SBML.MathApply("lt", [
+            SBML.MathVal{Float64}(1.0),
+            SBML.MathIdent("S1")
+        ]),
+        SBML.MathApply("lt", [
+            SBML.MathIdent("S1"),
+            SBML.MathVal{Float64}(100.0)
+        ])
+    ]),
+        "<message>\n  <p xmlns=\"http://www.w3.org/1999/xhtml\"> Species S1 is out of range. </p>\n</message>",
```

Comment:
I'm trying to play with the functions for the [`XMLNode_t`](http://sbml.org/Software/libSBML/5.18.0/docs/formatted/c-api/class_x_m_l_node__t.html) class but I still haven't figured out how to get only the message text, rather than the full XML node :thinking:

Reply:
Ok, thanks to hints from my colleagues I found the incantation to extract only the text from an XML node.  It may not be 100% reliable, but should work for the time being.  `listOfConstraints` doesn't seem to be widely used, so it may not be worth spending more time on this until we have an actual use case.